opsdroid/events.py

```diff
@@ -271,6 +270,26 @@ def __repr__(self):
         return await super().respond(response)
 
 
+class EditedMessage(Message):
+    """A  `opsdroid.events.Message` which has been edited.
+
+    The ``linked_event`` property should hold either an `opsdroid.events.Event`
+    class or an id for an event to which the edit applies.
+    """
+
+    def __init__(self, *args, **kwargs):
+        """Create object with minimum properties."""
+        super().__init__(*args, **kwargs)
+
+
+class Reply(Message):
```

Comment:
Should calling `Message.respond(string)` generate a `Reply` event instead of a `Message` event then?

Reply:
I would say no, because that would be a rather large change in behavior. I am seeing Reply here as also applying to slack threads, where you would reply to a message to put it in a thread.

Reply:
Ah that makes sense!